controller/webhook/server.go

```diff
@@ -160,7 +160,6 @@ func (s *Server) processReq(ctx context.Context, data []byte) *admissionv1beta1.
 		return admissionReview
 	}
 	log.Infof("received admission review request %s", admissionReview.Request.UID)
-	log.Debugf("admission request: %+v", admissionReview.Request)
```

Comment:
I removed this because it's not properly decoded and we just see tons of numbers. Even if properly decoded it doesn't seem it has useful data not available in the other log statements.